app/templates/views/user-already-invited.html

```diff
@@ -14,9 +14,16 @@
 
   {{ page_header('This person has already received an invite') }}
 
-  <p class="govuk-body">
-    {{ user_to_invite.name }} has not accepted their invitation to
-    ‘{{ current_service.name }}’ yet. You do not need to do anything.
-  </p>
+  {% if user_to_invite.name %}
+    <p class="govuk-body">
+      {{ user_to_invite.name }} has not accepted their invitation to
+      ‘{{ current_service.name }}’ yet. You do not need to do anything.
+    </p>
+  {% else %}
+    <p class="govuk-body">
+      User has not accepted their invitation to
+      ‘{{ current_service.name }}’ yet. You do not need to do anything.
+    </p>
+  {% endif %}
```

Comment:
I wonder if this would be more DRY? But I haven't tested it though!

```suggestion
  <p class="govuk-body">
    {{ (user_to_invite and user_to_invite.name) or 'User' }} has not accepted their invitation to
    ‘{{ current_service.name }}’ yet. You do not need to do anything.
  </p>
```